libs/comments/website/src/lib/comment-list/comment-list.tsx

```diff
@@ -1,17 +1,129 @@
+import {Modal, css, styled} from '@mui/material'
+import {LoginFormContainer, useUser} from '@wepublish/authentication/website'
+import {Button, IconButton, Link} from '@wepublish/ui'
 import {Comment} from '@wepublish/website/api'
 import {BuilderCommentListProps, useWebsiteBuilder} from '@wepublish/website/builder'
-import {MdForum} from 'react-icons/md'
+import {useMemo, useState} from 'react'
+import {MdClose, MdForum} from 'react-icons/md'
+import {Node} from 'slate'
 import {getStateForEditor} from './comment-list.state'
 
 export const CommentListWrapper = styled('section')`
   display: grid;
   gap: ${({theme}) => theme.spacing(4)};
 `
 
-export const CommentListActions = styled('div')``
+export const CommentListActions = styled('div')`
+  display: flex;
+  justify-content: end;
+`
+
+export const CommentListReadMore = styled(Button)`
+  padding: ${({theme}) => `${theme.spacing(1)} ${theme.spacing(2.5)}`};
+  text-transform: uppercase;
+`
+
+export const ButtonsWrapper = styled('div')`
+  width: auto;
+  margin-top: ${({theme}) => theme.spacing(5)};
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    flex-direction: row;
+    width: ${({theme}) => theme.spacing(86)};
+  }
+`
+
+export const LoginButton = styled(Button)`
+  margin-top: ${({theme}) => theme.spacing(3)};
+  border-width: 1px;
+
+  &:hover {
+    border-width: 1px;
+  }
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    margin-top: 0;
+    margin-left: ${({theme}) => theme.spacing(6)};
+  }
+`
+
+export const CommentEditorOuter = styled('div')`
+  padding: ${({theme}) => theme.spacing(4)};
+  width: 100%;
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    width: ${({theme}) => theme.spacing(80)};
+  }
+`
+
+export const LoginWrapper = styled('div')`
+  position: relative;
+  display: grid;
+  grid-template-columns: 1fr;
+  padding-top: ${({theme}) => theme.spacing(4)};
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    grid-template-columns: repeat(2, 1fr);
+  }
+`
+
+export const Register = styled('div')`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  font-size: 1.8rem;
+  margin: ${({theme}) => theme.spacing(4)} 0;
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    grid-column: 2/3;
+    margin: 0 ${({theme}) => theme.spacing(4)};
+  }
+`
+
+export const CloseLogin = styled(IconButton)`
+  position: absolute;
+  top: ${({theme}) => `-${theme.spacing(2)}`};
+  right: ${({theme}) => `-${theme.spacing(2)}`};
+`
+
+export const ModalContent = styled('div')`
+  position: absolute;
+  left: 50%;
+  transform: translateX(-50%);
+  padding: ${({theme}) => `${theme.spacing(4)} ${theme.spacing(3)}`};
+  bottom: 0;
+  background-color: ${({theme}) => theme.palette.common.white};
+  width: 100%;
+
+  ${({theme}) => theme.breakpoints.up('md')} {
+    width: auto;
+  }
+
+  animation: slideUpAndFadeIn 0.5s ease forwards;
+
+  @keyframes slideUpAndFadeIn {
```

Comment:
Should probably be standardized as it seems out of place here.
Can also use `keyframes` similar to styled or css.

Reply:
removed